src/TheThingsNetwork.cpp

```diff
@@ -201,7 +201,11 @@ bool TheThingsNetwork::enableFsbChannels(int fsb) {
   return true;
 }
 
-bool TheThingsNetwork::personalize(const byte devAddr[4], const byte nwkSKey[16], const byte appSKey[16]) {
+bool TheThingsNetwork::personalize(const byte devAddr[4], const byte nwkSKey[16], const byte appSKey[16], bool reset) {
```

Comment:
As per @johanstokking's comment https://github.com/TheThingsNetwork/arduino-device-lib/pull/42#issuecomment-247356579 we can drop the `reset` arg and add a `personalize()` proto with no args which does no `reset()` and skips the 3 `mac set` commands. Of course this means the `personalize()` which does take 3 arguments can start with `reset()` and the 3 commands and then call the `personalize()` with no arguments to keep code DRY.